# TODO: Remove this after gobject-introspection 1.84.0 update in host system

--- a/gtk/meson.build
+++ b/gtk/meson.build
@@ -1153,7 +1153,6 @@
   gir_args = [
     '-DGTK_COMPILATION',
     '--quiet',
-     '--doc-format=gi-docgen',
   ]
 
   gdk_gir_inc = [ 'cairo-1.0', 'Gio-2.0', 'GdkPixbuf-2.0', 'Pango-1.0', 'PangoCairo-1.0' ]
